chrome/options.browserify.js

```diff
@@ -9,6 +9,11 @@ var settings = {
     title: "Use fuzzy search",
     value: true
   },
+  filterEverything: {
+    type: "checkbox",
+    title: "Filter all available entries",
+    value: false
```

Comment:
I thought about it, let's have it enabled by default. Experiments show that the lag is not that big, and seeing the results as you type is much better for the user experience.